guide/src/redshift.md

```diff
@@ -35,3 +35,14 @@ This may require some experimentation.
 ```txt
 {{#include generated/features_redshift.txt}}
 ```
+
+## Example
+
+```
+dbcrossbar cp \
+  csv:mytable.csv \
+  redshift://myuser:XXXXXX@redshift-cluster-1.YYYYYYY.us-west-2.redshift.amazonaws.com:5439/mydb#mytable  \
+  --temporary=s3://mybucket-in-us-west-2/ \
+  --if-exists=upsert-on:id \
+  --to-arg=credentials="aws_access_key_id=$AWS_ACCESS_KEY_ID;aws_secret_access_key=$AWS_SECRET_ACCESS_KEY;token=$AWS_SESSION_TOKEN"
```

Comment:
Can you point me to the part of the manual that documents `credentials=`? I was under the vague impression that there was a different way to pass these that might be preferred?

Reply:
right, you can do it with an IAM role, i believe:

https://docs.aws.amazon.com/redshift/latest/dg/copy-parameters-authorization.html